Description: prefix in makefile
 Change the hardcoded PREFIX in the Makefile from /usr/local
 to /usr to satisfy the guidelines.
 .
 easystroke (0.5.6-1nano) unstable; urgency=low
 .
   * Initial release
Author: Christopher Roy Bratusek <nano@tuxfamily.org>
--- easystroke-0.5.6.orig/Makefile
+++ easystroke-0.5.6/Makefile
@@ -13,7 +13,7 @@
 #  CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 DESTDIR  =
-PREFIX   = /usr/local
+PREFIX   = /usr
 BINDIR   = $(PREFIX)/bin
 ICONDIR  = $(PREFIX)/share/icons/hicolor/scalable/apps
 MENUDIR  = $(PREFIX)/share/applications
